Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:7
+MAINTAINER Avishkar Gupta <avgupta@redhat.com>
+
+RUN yum install -y epel-release && \
+    yum install -y zip && \
+    yum install -y python-pip python-devel gcc && \
+    yum -y install tkinter && \
+    yum-config-manager --disable testing-devtools-2-centos-7 && \
+    yum -y install gcc-c++.x86_64 && \
```

Comment:
Can we install all the packages with one `yum` call ?
```
RUN yum -y install -y epel-release && \
    yum-config-manager --disable testing-devtools-2-centos-7 && \
    yum install -y zip python-pip python-devel gcc gcc-c++.x86_64 tkinter && \
    yum clean all
```

Why do we disable the `testing-devtools-2-centos-7` repo ?

Reply:
I took this from the Kronos Dockerfile we had, assuming it had a reason but didn't read, apologies for that. From what it looks like the reason might have something to do with installation of gcc_c++ like here: https://stackoverflow.com/questions/24634776/installing-multiple-gcc-on-centos-7. I tried a build with that line removed and it passed, so I'm assuming whatever issue was there has been fixed.

I'll remove this as well.